### PR TITLE
Linux support. Minor fix.

### PR DESCRIPTION
**Pull request by yuppox**
Added Linux Makefile.
Added os.h.
Added STRCMP macros.
Added missing limits.h include.

### CODE REVIEW

File: os.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Legacy BASIC
+// Utility types and functions.
+// Copyright (c) 2021-2 Nigel Perks
```
Comment (nigelperks) [inline os.h:3]: Is that copyright OK? I'm happy if you want to put "contributed by ...." or similar.

Comment (nigelperks) [inline os.h:3]: Is that copyright OK? I'm happy if you want to put "contributed by ...." or similar.

File: os.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+#ifndef OS_H
+#define OS_H
+
+#ifdef LINUX
+#define __STRICMP strcasecmp
+#define __STRNICMP strncasecmp
```
Comment (nigelperks) [inline os.h]: I believe identifiers beginning with two underscores (or an underscore and capital letter) are reserved in ISO standard C. Can we change it to just STRICMP ?

Comment (nigelperks) [inline os.h]: I believe identifiers beginning with two underscores (or an underscore and capital letter) are reserved in ISO standard C. Can we change it to just STRICMP ?

**Review by nigelperks — CHANGES_REQUESTED** [review]
Thanks for this! It built on Ubuntu and I'm happy to take it if we can fix the reserved identifier. I'll add "Linux build contributed by yuppox" somewhere unless you prefer me not to or you would like your name instead.

File: os.h (hunk 1 above)
Comment (yuppox) [reply os.h:3]: Yes, that copyright is fine. :)

File: os.h (hunk 2 above)
Comment (yuppox) [reply os.h]: That could be. I agree it's best to change it and be sure.

**Comment by yuppox** [comment]
Macro updated.

**Review by nigelperks — APPROVED** [review]
Thanks for finding the program and adding Linux support.

**Pull request merged**